sw/device/tock/README.md

```diff
@@ -0,0 +1,52 @@
+# Tock OS
+
+Tock OS is a secure embedded operating system for microcontrollers.
+
+This is not a complete Tock OS implementation. OpenTitan relevant contents of
+`board` and `chip` directories are implemented and maintained downstream, the
+other Tock OS dependencies are pulled from the upstream or local Tock OS repo.
+Please see [Rationale](# Rationale) and [Directory layout](# Directory layout)
```

Comment:
These links don't render. It's a short doc, so I'd just say "See below for more details." or similar.

Reply:
Done.